bin/reload-upstream

```diff
@@ -15,6 +15,7 @@ function fetchdocs () {
   LOCAL_PATH="$1"
   GIT_REPO="$2"
   GIT_PATH="$3"
+  GIT_BRANCH="$4"
```

Comment:
You should either default this to `master` (so it's not required and as git's default value) or also update the [respective docs page](https://github.com/logsearch/logsearch-website/blob/master/docs/website/guides/managing-the-upstream-docs.md) to indicate it's a required parameter.